stream-chat-android-client/api/stream-chat-android-client.api

```diff
@@ -1954,7 +1954,14 @@ public final class io/getstream/chat/android/client/models/Attachment$UploadStat
 }
 
 public final class io/getstream/chat/android/client/models/Attachment$UploadState$InProgress : io/getstream/chat/android/client/models/Attachment$UploadState {
-	public static final field INSTANCE Lio/getstream/chat/android/client/models/Attachment$UploadState$InProgress;
+	public fun <init> (J)V
+	public final fun component1 ()J
+	public final fun copy (J)Lio/getstream/chat/android/client/models/Attachment$UploadState$InProgress;
+	public static synthetic fun copy$default (Lio/getstream/chat/android/client/models/Attachment$UploadState$InProgress;JILjava/lang/Object;)Lio/getstream/chat/android/client/models/Attachment$UploadState$InProgress;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getCurrentProgress ()J
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
```

Comment:
Changing `InProgress` from an object to a data class is a breaking change, should definitely be in the changelogs